### ISSUE TITLE
Retrieve events over multiple missing events boundary

### ISSUE THREAD

**Issue by uhppoted** [uhppoted/uhppoted-httpd#8]
There appears to be a bug in the UHPPOTE controller firmware around power on events:

If _record special events_ is enabled, at power-on the controller should record a POWER ON event. It probably does because the event is viewable in the manufacturer _AccessControl_ application event logs, but retrieving the event at the expected index using the `get-event` function in the public API returns 'no event':
```
 ... request
 ...          00000000  17 b0 00 00 2d 55 39 19  2c 28 03 00 00 00 00 00  |....-U9.,(......|
 ...          00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          00000020  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          00000030  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          

 ... response
 ...          00000000  17 b0 00 00 2d 55 39 19  00 00 00 00 00 00 00 00  |....-U9.........|
 ...          00000010  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          00000020  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          00000030  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 ...          

   ERROR: 423187757:  no event at index: 206892
```

Ordinarily, the _uhppoted-httpd_ event retrieval logic will skip the missing event and continue fetching any events after the missing _POWER ON_ event - except in the case where there are 5 or more consecutive missing events in which case it gives up forever.

Ref. https://github.com/uhppoted/uhppoted/discussions/13#discussioncomment-3111298

**Comment by twystd** [comment]
Fixed (along with the missing onMore handler for scrolling through events).

Pending release.

**Comment by uhppoted** [comment]
Fixed in [release v0.8.1](https://github.com/uhppoted/uhppoted-httpd/releases/tag/v0.8.1)
